.github/workflows/goTest.yml

```diff
@@ -121,16 +122,16 @@ jobs:
           GOEXPERIMENT: nocoverageredesign
       -
         name: Annotate Test Suite Results
-        if: ${{ (success() || failure()) && hashFiles('gotestsum.json') != '' }}
+        if: ( success() || failure() ) && hashFiles('gotestsum.json') != ''
         uses: guyarb/golang-test-annotations@2941118d7ef622b1b3771d1ff6eae9e90659eb26 # v0.8.0
         with:
           test-results: gotestsum.json
       -
         name: Codecov
         uses: codecov/codecov-action@13ce06bfc6bbe3ecf90edbbf1bc32fe5978ca1d3 # v5.3.1
-        if: inputs.run-codecov && matrix.go == 'stable'
+        if: inputs.run-codecov
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          files: ./coverage.out # optional
-          name: codecov-umbrella # optional
-          fail_ci_if_error: false # optional (default = false)
+          files: ./coverage.out
+          name: go${{ steps.setup-go.outputs.go-version }}-${{ runner.os }}
+          fail_ci_if_error: true
```

Comment:
Can it become opt-in per repo for now? My fear is that switching it on org-wide at once could cause some annoyance without having tried it on a few first.

Is there a(nother) workaround for hotfixes?